### PR TITLE
po: Add Tamil translation

### PR DESCRIPTION
**Pull request by kbdharun**
Reopened PR, Sorry for the error in the previous PR.
Ref: #469

### CODE REVIEW

**Comment by kbdharun** [comment]
Is it fine now @tchx84?

File: po/ta.po [hunk 1]
```diff
@@ -0,0 +1,575 @@
...
+"Flatseal is a graphical utility to review and modify permissions from your Flatpak "
+"applications."
+msgstr ""
+"பிளாட்சீல் என்பது உங்கள் பிளாட்பாக் பயன்பாடுகளின் அனுமதிகளை மதிப்பாய்வு செய்வதற்கும் "
+"மாற்றியமைப்பதற்கும் ஒரு வரைகலைப் பயன்பாடாகும்."
+
+#. TRANSLATORS: Don't translate this text
+#: data/com.github.tchx84.Flatseal.appdata.xml.in:31
+msgid "Martin Abente Lahaye"
+msgstr "மார்டின் அபென்டே லஹயே"
```
Comment (tchx84) [inline po/ta.po]: This string is still translated :thinking:

File: po/ta.po [hunk 2]
```diff
@@ -0,0 +1,575 @@
...
+msgid "_Documentation"
+msgstr "_ஆவணப்படுத்தல்"
+
+#: src/widgets/menu.ui:15
+msgid "_Keyboard Shortcuts"
+msgstr "_விசைப்பலகை குறுக்குவழிகள்"
+
+#: src/widgets/menu.ui:19
+msgid "_About Flatseal"
+msgstr "_ பிளாட்சீல் பற்றி"
```
Comment (tchx84) [inline po/ta.po:357]: By any chance, have you tested if these underscores do actually work? e.g. 

1. Open Flatseal with your translation.
2. Open the main menu
3. Long press Alt
4. Check if the menu items are highlighted for keyboard activation
5. Check if the menu items can be activated with the keyboard

**Review by tchx84 — CHANGES_REQUESTED** [review]
Almost there!

File: po/ta.po (hunk 1 above)
Comment (kbdharun) [reply po/ta.po]: Oh, Sorry, I used Poedit during translation, I didn't notice the messages.I will review it once again and make the necessary changes.

File: po/ta.po (hunk 2 above)
Comment (kbdharun) [reply po/ta.po:357]: I will give it a try and I will update it here.

**Comment by tchx84** [comment]
@kbdharun let me know if you're having troubles testing it, you should only include the .po files. The .mo files are created when the application builds.

File: po/ta.po [hunk 3]
```diff
@@ -0,0 +1,575 @@
...
+msgstr "பயன்பாட்டிற்கு ஏற்றுமதி செய்யப்பட்ட மாறிகளின் பட்டியல்"
+
+#: src/widgets/aboutDialog.js:38
+msgid "Contributions by"
+msgstr "மூலம் பங்களிப்புகள்"
+
+#. TRANSLATORS: <full-month-name> <day-of-month>, <year-with-century>
+#: src/widgets/appInfoViewer.js:61
+msgid "%B %e, %Y"
+msgstr "%B %e, %Y"
```
Comment (kbdharun) [inline po/ta.po:292]: Regarding this string is it fine or should I make it into `<full-month-name> <day-of-month>, <year-with-century>` 's Tamil equivalent?

**Comment by kbdharun** [comment]
> @kbdharun let me know if you're having troubles testing it, you should only include the .po files. The .mo files are created when the application builds.

Yeah, I will try it now and I will let you know. Yeah, I don't include `.mo` files usually too. (Removed it)

**Comment by kbdharun** [comment]
Update: The build succeeded in `gnome-builder`. I will test it out and I will update it here in 10 minutes.
image

**Comment by kbdharun** [comment]
Hey, I exported the bundle and installed the flatpak but I am unable to see the Tamil translation from ta.po file even if I am in ta locale in the system.

Should I build it manually? Can you assist me here?

**Comment by tchx84** [comment]
@kbdharun oh, one thing I noticed it's missing, is that you need to add your language code to po/LINGUAS file, and commit the change.

Re. testing this:
1. Make sure your changes are committed to your branch.
2. While on your branch, build Flatseal following these steps https://github.com/tchx84/Flatseal#build-it-yourself
3. Run Flatseal with `LC_ALL=ta_IN.UTF-8 flatpak --user run --branch=master com.github.tchx84.Flatseal`

**Comment by kbdharun** [comment]
What Platform do I choose, I am running on `Fedora Workstation 36` in GNOME 42.

I am getting this error

```bash
[dharunkrishna@dharun-fedora-laptop Flatseal]$ flatpak --user install org.gnome.{Platform,Sdk}
Looking for matches…
error: No remote refs found similar to ‘org.gnome.Platform’
```

**Comment by tchx84** [comment]
> What Platform do I choose, I am running on `Fedora Workstation 36` in GNOME 42.
> 
> I am getting this error
> 
> ```shell
> [dharunkrishna@dharun-fedora-laptop Flatseal]$ flatpak --user install org.gnome.{Platform,Sdk}
> Looking for matches…
> error: No remote refs found similar to ‘org.gnome.Platform’
> ```

It seems you don't have Flathub installed? Try this

```
flatpak --user remote-add --if-not-exists flathub https://flathub.org/repo/flathub.flatpakrepo
```

**Comment by kbdharun** [comment]
Installed it thanks, I will test it out and will reply soon.

**Comment by kbdharun** [comment]
Update: Everything works perfectly 🎉 ❤️ @tchx84 

## Images for reference:

Screenshot from 2022-09-28 19-16-40

Screenshot from 2022-09-28 19-16-40

Screenshot from 2022-09-28 19-16-55

Screenshot from 2022-09-28 19-17-05

Screenshot from 2022-09-28 19-17-17

Screenshot from 2022-09-28 19-17-28

Screenshot from 2022-09-28 19-17-36

Screenshot from 2022-09-28 19-17-42

Screenshot from 2022-09-28 19-18-08

**Comment by kbdharun** [comment]
I have some questions
1. Can the documentation section be translated too in the future?
2. Also just out of curiosity, Are there any plans to port `Flatseal` to **GTK 4**?

**Comment by tchx84** [comment]
> Update: Everything works perfectly tada heart @tchx84

Excellent! Last ask, can you unify all commits (except the aboutDialog one) into the first commit? So we only have 2 commits like before.

**Comment by tchx84** [comment]
> I have some questions
> 
> 1. Can the documentation section be translated too in the future?

Yeah, it's in my plans. But it will also depends on translators time, of course.

> 2. Also just out of curiosity, Are there any plans to port `Flatseal` to **GTK 4**?

Yup, there's already a PR for that. but, there's still a few issues to be solved before we can merge it.

**Comment by kbdharun** [comment]
> > Update: Everything works perfectly tada heart @tchx84
> 
> Excellent! Last ask, can you unify all commits (except the aboutDialog one) into the first commit? So we only have 2 commits like before.

Is it like `git rebase`? Can you tell me how to do it.

**Comment by tchx84** [comment]
> > > Update: Everything works perfectly tada heart @tchx84
> > 
> > 
> > Excellent! Last ask, can you unify all commits (except the aboutDialog one) into the first commit? So we only have 2 commits like before.
> 
> Is it like `git rebase`? Can you tell me how to do it.

try this..

```
git rebase -i a56fc8756d3f2d6671a4133fe0f8a039ba9d475f~
# then move the last 3 commits under the first one, and mark the ones you moved as "fixup".
```

After that, you should have only 2 commits, the first one (with all the changes under po/) and then the second commit with the change to aboutDialog.

**Comment by kbdharun** [comment]
Done 🎉 , Thanks for your insights.

**Review by tchx84 — APPROVED** [review]

**Comment by tchx84** [comment]
@kbdharun ty!

**Pull request merged**